.github/workflows/readme-check.yml

```diff
@@ -0,0 +1,35 @@
+name: Check for README.md and LISEZMOI.md
+
+on:
+  push:
+    branches: [action-trigger]
+  workflow_dispatch:
+
+jobs:
+  readme-check:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Get list of repos
+      env:
+        TOKEN: ${{ secrets.MY_PAT }}
+      run: |
+        result=`curl -H "Authorization: $TOKEN" -H "Accept: application/vnd.github.v3+json" https://api.github.com/orgs/OpenDRR/repos?per_page=100 | jq .[].name | tr -d \"`
+        results=($result)
+        for REPO in "${results[@]}"
+        do
+          RESPONSE=`curl -i https://api.github.com/repos/OpenDRR/$REPO/contents/README.md --write-out "%{http_code}\n" --output /dev/null --silent`
+          FRESPONSE=`curl -i -H "Authorization: $TOKEN" https://api.github.com/repos/OpenDRR/$REPO/contents/LISEZMOI.md --write-out "%{http_code}\n" --output /dev/null --silent`
+          if [ $RESPONSE == 404 ] && [ $FRESPONSE == 404 ]
```

Comment:
I would suggest using double square brackets, i.e. `[[ $RESPONSE == 404 ]] && [[ $FRESPONSE == 404 ]]` as otherwise the variables need to be double-quoted.  In ShellCheck's words: SC2086 (info): Double quote to prevent globbing and word splitting.